internal/fileutils/fileutils.go

```diff
@@ -772,6 +772,26 @@ func ResolvePath(path string) (string, error) {
 	return evalPath, nil
 }
 
+// ResolveUniquePath gets the absolute location of the provided path
+// with the best effort attempt to produce the same result for all possible paths to the
+// given target.
+func ResolveUniquePath(path string) (string, error) {
+	evalPath, err := ResolvePath(filepath.Clean(path))
```

Comment:
Wouldn't it be safer to put `filepath.Clean()` at the end of this function? Otherwise we're assuming both ResolvePath and GetLongPathName will only ever produce cleaned paths themselves.

Reply:
I fundamentally agree and had it like this in the beginning, but then on MacOS the unit tests fail.  I don't understand it still, although I put quite some effort in trying to understand it, but if I put `/private/var/temp/./targetdir` in there, `ResolveUniquePath` does not resolve the symlink to `/var/temp/targetdir`.  If I put the cleaned version `/private/var/temp/targetdir` in, it does.

Cf., https://github.com/ActiveState/cli/runs/1289415663#step:7:37

Of course, I could also add a `filepath.Clean` at the end again.  Maybe I'll do that, just to be (more) sure.

Reply:
Yeah I think that works; call it twice. 

I remember struggling with the macOS tempdir issue way back, never quite figured it out either.